example/example.wsgi

```diff
@@ -0,0 +1,150 @@
+# This file is part of Flask-MultiAuth.
```

Comment:
:+1: I was just feeling dirty for committing this whole file that it's pretty much a copy of the other.